packages/data-api/src/EventsFetchQuery.js

```diff
@@ -0,0 +1,78 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2021 Beyond Essential Systems Pty Ltd
+ */
+
+import { DataFetchQuery } from './DataFetchQuery';
+
+export class EventsFetchQuery extends DataFetchQuery {
```

Comment:
Great job breaking out the queries between `fetchAnalytics` and `fetchEvents`! Trying to support one in the same queryBuilder was getting pretty untenable